src/main/java/yapper/exceptions/ExceptionHandler.java

```diff
@@ -0,0 +1,231 @@
+package yapper.exceptions;
+
+import yapper.io.StringStorage;
+
+/**
+ * The {@code ExceptionHandler} class provides methods for validating user input and
+ * handling exceptions related to task management in the Yapper chatbot application.
+ *
+ * <p>
+ * It throws {@code YapperException} with appropriate messages if any checks below fail:
+ * <li> empty or unknown instructions <li/>
+ * missing arguments and keywords <li/>
+ * invalid task types and statuses <li/>
+ * <p/>
+ *
+ */
+public class ExceptionHandler {
+
+    // User Input Validation Errors
+
+    /**
+     * Checks if the string is empty when reading from user.
+     *
+     * @param userInputString the input string to validate
+     * @param isAboutArgs flag indicating if the check is for arguments
+     * @throws YapperException if the input string is empty
+     */
+    public static void checkIfUserInputEmpty(String userInputString, boolean isAboutArgs) throws YapperException {
```

Comment:
can this be 2 separate methods - one while reading the file and the other while reading input from user?